lib/screens/edit_tiles_screen.dart

```diff
@@ -1,246 +1,246 @@
-            SizedBox(
-              height: screenSize.height / 30,
-            ),
-            // BUTTONS
-            Row(
-              children: <Widget>[
-                SizedBox(
-                  width: screenSize.width / 30,
-                ),
-                Expanded(
-                  child: RaisedButton(
-                    color: Theme.of(context).primaryColor,
-                    shape: RoundedRectangleBorder(
-                        borderRadius: BorderRadius.circular(6.0)),
-                    child: Text(
-                      "Lock",
-                      style: TextStyle(fontSize: 18.0, color: Colors.white),
-                    ),
-                    onPressed: () {
-                      setState(() => lock = !lock);
-                    },
-                  ),
-                ),
-                SizedBox(
-                  width: screenSize.width / 30,
-                ),
-                Expanded(
-                  child: RaisedButton(
-                    color: Color(0xFF3F51B5),
-                    shape: RoundedRectangleBorder(
-                        borderRadius: BorderRadius.circular(6.0)),
-                    child: Text(
-                      "Hide",
-                      style: TextStyle(fontSize: 18.0, color: Colors.white),
-                    ),
-                    onPressed: () => print('Hide'), // debug: add function
-                  ),
-                ),
-                SizedBox(
-                  width: screenSize.width / 30,
-                ),
-                Expanded(
-                  child: RaisedButton(
-                    color: Color(0xFFBB0000),
-                    shape: RoundedRectangleBorder(
-                        borderRadius: BorderRadius.circular(6.0)),
-                    child: Text(
-                      "Delete",
-                      style: TextStyle(fontSize: 18.0, color: Colors.white),
-                    ),
-                    onPressed: () => deleteTile(context),
-                  ),
-                ),
-                SizedBox(
-                  width: screenSize.width / 30,
-                ),
-              ],
-            )
-          ],
-        ),
-      ),
-    );
-  }
-}
+import 'package:cboard_mobile/widgets/widgets.dart';
+import 'package:flutter/material.dart';
+
+class EditTileScreen extends StatefulWidget {
+  @override
+  _EditTileScreenState createState() => _EditTileScreenState();
+}
+
+class _EditTileScreenState extends State<EditTileScreen> {
+  bool lock = false;
+  @override
+  Widget build(BuildContext context) {
+    final Size screenSize = MediaQuery.of(context).size;
+
+    return Scaffold(
+      appBar: AppBar(
+        leading: GestureDetector(
+          onTap: () => print('Back'),
+          child: Icon(Icons.arrow_back, color: Colors.white, size: 35.0),
+        ),
+        centerTitle: true,
+        title: Text(
+            'Edit (5 tiles)'), // Debug: Change title to keep track of how many tiles
+      ),

```

Comment:
I made a shared `AppBar` component, so let's use that here instead! I think it simplifies much of the back navigation automatically!